features/retry.feature

```diff
@@ -570,3 +570,61 @@ Feature: Retry flaky tests
       And scenario "Failing" attempt 0 step "Given a failing step" has status "failed"
       And scenario "Failing" attempt 1 step "Given a failing step" has status "failed"
       And scenario "Passing" step "Given a passing step" has status "skipped"
+
+  Scenario: RerunFormatter does not report attempts that are retried
```

Comment:
Could we not achieve the test by using the rerun formatter directly and observing the file that it writes, rather than wrapping it in a custom formatter? I don't doubt that it works, but I wouldn't want a user to take this example and run with it.

Reply:
That's a neat feature. :) I was not aware that I can simply reproduce with `--format rerun`. The changeset became much simpler.